tools/alpine/versions.aarch64

```diff
@@ -1,4 +1,4 @@
-# linuxkit/alpine:3f3f1159e07077cdcf1544f536cfee5052fffa89-arm64
+# linuxkit/alpine:4967a81f63c6fab8cfa686ba051ebe8e0f0fd8ee-
```

Comment:
Forgive my ignorance :) is this a generated list of package versions in the image? Is it expected that the `arm64` suffix is removed?

Reply:
Yeah, this has now been fixed with https://github.com/linuxkit/linuxkit/pull/3665. I'll merge this anyway, since it's mostly cosmetic.